Clone news-week

Luckyaremu opened the pull request:
Requesting for code review for the solo bootstrap project of news-week pages

Luckyaremu committed:
html completed

Luckyaremu committed:
stylelint errors fixed

Luckyaremu committed:
project done for submission for review

euqueme commented on README.md:
```diff
@@ -1 +1,74 @@
+
+![screenshot](img\screenshot.jpg)
+
+A clone of the beautifull newsweek page
```
on line #5 is "beautiful" not "bautifull" let's correct it

euqueme commented on README.md:
```diff
@@ -1 +1,74 @@
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
```
remove these instructions from the readme template ad they have no meaning for the project

euqueme commented on README.md:
```diff
@@ -1 +1,74 @@
+- Install Node JS and nmp(most recents versions)
+- Run this command on your terminal in order to install stylelint: npm install stylelint stylelint-config-recommended --save-dev
+
+### Usage
```
remove the Install and Usage titles as you don't need to install VS code to run this project lines #32 to #38

euqueme commented on README.md:
```diff
@@ -1 +1,74 @@
\ No newline at end of file
+# newsweek-clone
+
+![screenshot](img\screenshot.jpg)
```
replace line #3 with the following to fix the broken image

`![screenshot](img/screenshot.jpg)`

euqueme reviewed (changes requested):
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Hello @Luckyaremu :handshake: 

### You have done an incredible work :medal_sports:  your page looks very similar to the original and you have used bootstrap very well :ok_hand: 

### There is some work to do on the screen breakpoints and general aspects of the project :point_down: 

## Small Screens

**All articles**

- [ ] The text content (mainly the bold headings) from all the articles are too big in font please reduce it 

- Yours

![image](https://user-images.githubusercontent.com/52209067/74954913-7a15c380-53c9-11ea-8e50-99582767fec3.png)

- Original

![image](https://user-images.githubusercontent.com/52209067/74955004-9ca7dc80-53c9-11ea-9363-5f9ed3a6aa01.png)

**Section Positioning**

- [ ] please follow the structure for the first 4 sections in this screen size:

1. TOP STORY
2. MORE STORIES
3. CULTURE & TRAVEL
4. FEATURED STORIES

- Yours

![image](https://user-images.githubusercontent.com/52209067/74952741-4be2b480-53c6-11ea-9161-0eb1a1c00b47.png)

- Original

![image](https://user-images.githubusercontent.com/52209067/74952873-851b2480-53c6-11ea-83c3-adb672fb047e.png)

**FEATURE STORIES**

- [ ] the articles images and the header should be smaller in height and size respectively

- Yours

![image](https://user-images.githubusercontent.com/52209067/74954261-6f0e6380-53c8-11ea-9a58-97dc01d243a8.png)

- Original

![image](https://user-images.githubusercontent.com/52209067/74954310-851c2400-53c8-11ea-9e20-ffd392e6f952.png)

**CULTURE & TRAVEL**

- [ ] each article should take 100% of the screen width

- Yours

![image](https://user-images.githubusercontent.com/52209067/74955301-117b1680-53ca-11ea-96f8-34ad4d2c9ebb.png)

- Original

![image](https://user-images.githubusercontent.com/52209067/74955346-2657aa00-53ca-11ea-8f81-5def01b9bb98.png)

**MORE STORIES, LATEST NEWS, and the aside articles for the U.S, World, Business, Tech & Science, Culture and Sports**

- [ ] The image for the articles of these sections should take 60% of the screen width and the content only 40%

- Yours

![image](https://user-images.githubusercontent.com/52209067/74955532-64ed6480-53ca-11ea-82b5-563e6da8d936.png)

- Original

![image](https://user-images.githubusercontent.com/52209067/74955668-9b2ae400-53ca-11ea-8360-520eb97763c8.png)

**IN THE MAGAZINE, EDITOR'S PICK and FEATURED SLIDE SHOW**

- [ ] this sections look good enough but you have to make them fit on the page, no horizontal scroll should appear

- Yours

![image](https://user-images.githubusercontent.com/52209067/74956168-53f12300-53cb-11ea-8cbf-a8af2c28c77a.png)

- Original

![image](https://user-images.githubusercontent.com/52209067/74956263-77b46900-53cb-11ea-8feb-a8541fa11682.png)

**SUBSCRIBE and NEWSLETTER**

- [ ] the images and content should be centered on the page yours are justified at the left (and overlapping at some points) also the NEWSLETTER image should take 100% of the screen width

- Yours

![image](https://user-images.githubusercontent.com/52209067/74957002-88b1aa00-53cc-11ea-82cf-eb88a1704ae9.png)

- Original

![image](https://user-images.githubusercontent.com/52209067/74957059-a1ba5b00-53cc-11ea-8a5b-6665d21419a1.png)

**Footer**

- [ ] the copyright and social icons should come after the NEWSWEEK logo in separate rows

- Yours

![image](https://user-images.githubusercontent.com/52209067/74957276-f1992200-53cc-11ea-91c3-75189fa41bc4.png)

- Original

![image](https://user-images.githubusercontent.com/52209067/74957317-ff4ea780-53cc-11ea-858f-42a22b61f917.png)

## Small and Medium Screens

**Navbar**

- [ ] The logo has to be smaller, currently is taking the whole screen making your page has a horizontal scroll also separate the SIGN IN link from the SUBSCRIBE >  button (do this also for the Big screens)

- Yours

![image](https://user-images.githubusercontent.com/52209067/74957649-784dff00-53cd-11ea-8c13-2b30cb024c0f.png)

- Original

![image](https://user-images.githubusercontent.com/52209067/74957675-84d25780-53cd-11ea-80aa-5af4862a0036.png)

**U.S, World, Business, Tech & Science, Culture and Sports**

- [ ] These sections should also appear in these breakpoints but they are currently hidden

- Yours

![image](https://user-images.githubusercontent.com/52209067/74956588-f27d8400-53cb-11ea-9bd0-e49cf5ca68e7.png)

- Original

![image](https://user-images.githubusercontent.com/52209067/74956635-05905400-53cc-11ea-95cc-a51e65140594.png)

## Medium Screens

**LATEST NEWS**

* [ ] the articles are overlapping let's fix them!
* [ ] the image for the articles is too big it should take only 20% of the screen width and the content 80%

- Yours

![image](https://user-images.githubusercontent.com/52209067/74958168-278ad600-53ce-11ea-83bf-0e7d5f00b944.png)

- Original

![image](https://user-images.githubusercontent.com/52209067/74958297-5739de00-53ce-11ea-96cd-d56a773d11cf.png)

-NEWSLETTER

the image should take 20% of the screen width and the content 80%

- Yours

![image](https://user-images.githubusercontent.com/52209067/74958472-9c5e1000-53ce-11ea-8415-4dea4d1a887a.png)

- Original

![image](https://user-images.githubusercontent.com/52209067/74958522-b26bd080-53ce-11ea-8906-8896aae47870.png)

## General

**OPINION**

- [ ] the color of the **OPINION** title should be light gray 

**Project repository**

- [ ] Edit your pull request description and add more to it to make it more professional
- [ ] I have added some suggestions in your README file please address them :arrow_down_small: 

### That's it! :raised_hands: 

### Please :pray: do the changes :memo: and submit back for a code review when you are done :rocket: 

Happy coding! :woman_technologist: 

Any question? reach to me on slack **@ Maru**

PROJECT STATUS: REQUESTED CHANGES :traffic_light:

Luckyaremu committed:
solving conflicts suggested by the TSE

Luckyaremu committed:
solved all the suggestions from tse

Luckyaremu committed:
comflicts solved

Luckyaremu committed:
code fixed all the issues recommended by tse

Luckyaremu committed:
read me updated

Luckyaremu committed:
read me updated

Luckyaremu committed:
screenshot updated

Luckyaremu commented:
Made changes required
1. Changes the font size
2.resized the containers
3. fixed the footer.
4. updated the readme and screenshot.
5. makes the project looks more presentable

pelzolga123 reviewed (changes requested):
Project requires changes.

pelzolga123 commented:
## Hi Lucky! ![](https://user-images.githubusercontent.com/38801423/62908122-703d4480-bd44-11e9-9115-42025ab7f341.png) 
### Great job so far :+1: 

Still there are some more changes required, before this pull request can be approved:
* [ ] Pull Request should have a short description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
### Mobile version
* [ ] As @euqueme  pointed earlier,  bold headings are too big, please make them smaller, so that your project will look more like original.

### Mobile and Tablet versions
* [ ] Small horizontal scrollbar is still present, that should be fixed:
![image](https://user-images.githubusercontent.com/20416431/75077246-db917b80-550a-11ea-87e9-e92b6a547a22.png) 

* [ ] Next part looks a bit messed up, please fix that:
![image](https://user-images.githubusercontent.com/20416431/75077387-3cb94f00-550b-11ea-8584-c12399016a70.png)
![image](https://user-images.githubusercontent.com/20416431/75077400-4b076b00-550b-11ea-923e-7358beb56d2c.png)

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

Luckyaremu committed:
conflicts resolved

Luckyaremu committed:
width, and image size fixed

Luckyaremu commented:
# Cloned Newsweek website home page

1. Image width solved
2. horizontal bar removed
3. font size reduced

mosesogwo reviewed (changes requested):
**Hi lucky, good work improving your project:clap:, but**

1. You haven't fixed the horizontal scroll earlier pointed out. Try to find out which section of the page or sizing is causing it and fix it.

2.  The top story section on small screen doesn't look good anymore.
![image](https://user-images.githubusercontent.com/46611870/75091280-ddd7f200-556b-11ea-9bb5-785256ba49f2.png)

Please, take the image out of the parent div to fix this issue. The image should be a direct child of the card class. Check the bootstrap doc for Cards and ensure you use it properly. Same goes for the Editor's picks section.

**Optional:** At around 759px, your More Stories section doesn't look good. You can try to fix that also.
![image](https://user-images.githubusercontent.com/46611870/75091341-6787bf80-556c-11ea-80c9-e0eda2bbbf67.png)

_**Kindly attend to the above and submit another code review request.**_

**Good luck:stars:**

Luckyaremu committed:
horizontal line removed, order conflicts resolved

Luckyaremu commented:
# Cloned Newsweek website home page
1. horizontal bar removed
2. div more image width fixed

davidmrtz-dev commented:
Pull Request under review by  **[TSE] David Elí**. 👨‍💻

davidmrtz-dev commented:
### **Status: Changes required.** 🛑 👀 🔍 
****
### Hi Team. 😄 
### Very good job so far, I notice you do a great job implementing the changes required by the previous TSE 🎊 
**Before move on let's solve a couple of issues.**  💯 👨‍💻  

****

- [ ] **Horizontal scroll on medium size devices.**

As the previous TSEs mentioned, on medium-sized devices there is a horizontal scroll bar that should not be there.

![Screen Shot 2020-02-22 at 3 34 14 PM](https://user-images.githubusercontent.com/52430951/75099597-08847380-5589-11ea-854f-851f0f92dab6.png)
 (image from [bootstrap doc](https://getbootstrap.com/docs/4.1/layout/overview/)).

This is how it looks. ⬇️ 

![Screen Shot 2020-02-22 at 3 26 26 PM](https://user-images.githubusercontent.com/52430951/75099640-80529e00-5589-11ea-81cd-f0698deb9f4a.png)

Here's how it looks on the original web page. ⬇️ 

![Screen Shot 2020-02-22 at 3 26 31 PM](https://user-images.githubusercontent.com/52430951/75099654-b55ef080-5589-11ea-9875-cbda59955857.png)
****
### Optional:

- [ ] **Section improve.**
In the small device's size. You can improve your divs to look more similar to the original page. 😄 💯 

This is how it looks. ⬇️ 

![Screen Shot 2020-02-22 at 3 21 52 PM](https://user-images.githubusercontent.com/52430951/75099747-e25fd300-558a-11ea-9b45-6011cacf8a17.png)

Original Page. ⬇️ 

![Screen Shot 2020-02-22 at 3 22 04 PM](https://user-images.githubusercontent.com/52430951/75099751-f1468580-558a-11ea-8675-60e768cabc1d.png)

****

### **Please kindly make the changes required and submit a new code review once you are ready!** 🙏 

Happy coding! 😄 💻  
If you have any questions or need help, you can message me on slack (@ David Eli Martinez Garcia).

**[TSE:  David Elí]** 👨‍💻 
Github: [@davitomix](https://github.com/davitomix)
Twitter: [@davitomix](https://twitter.com/davitomix007)
Linkedin: [David Elí Martínez García](https://www.linkedin.com/in/davideligarcia/)

Luckyaremu committed:
horizontal bar fixed on all widths

Luckyaremu commented:
Cloned Newsweek website home page
1. horizontal bar removed from all width
2. the more image and text flex throughout the page depending on the width, at less than 600px, some text goes under the image and at 600px above they stand alone which is also like the original.
![one](https://user-images.githubusercontent.com/37473601/75100512-c4b45d00-55ce-11ea-9d08-736f64374e4d.jpg)

davidmrtz-dev commented:
### **Status: Approved.** ✅ 
### **Congragulations!** 🎉 
****
### Hi Team. 😄 

Great Job 💯  @Luckyaremu  @mosesogwo , you can move onto the next project. 🎊 
Happy coding! 😄 💻  

**[TSE:  David Elí]** 👨‍💻 
Github: [@davitomix](https://github.com/davitomix)
Twitter: [@davitomix](https://twitter.com/davitomix007)
Linkedin: [David Elí Martínez García](https://www.linkedin.com/in/davideligarcia/)
